ontopy/ontology.py

```diff
@@ -353,23 +362,34 @@ def get_by_label(
 
         if colon_in_label is None:
             colon_in_label = self._colon_in_label
-        if colon_in_label:
-            if prefix:
-                raise ValueError(
-                    "`prefix` cannot be combined with `colon_in_label`"
+        if colon_in_label and prefix:
+            raise ValueError(
+                "`prefix` cannot be combined with `colon_in_label`"
+            )
+
+        splitlabel = label.split(":", 1)
+        if len(splitlabel) == 2 and not splitlabel[1].startswith("//"):
```

Comment:
The case where `splitlabel[1]` starts with "//" seems not to be handled. E.g. `label="http://xyz.com"`. This case should probably result in an error.